client/src/components/Explorer/__snapshots__/ExplorerHeader.test.js.snap

```diff
@@ -46,7 +20,7 @@ exports[`ExplorerHeader #page 1`] = `
       title={null}
     />
     <span>
-      test
+      Pages
```

Comment:
This looks like a bug, but most likely only because the test case needs updating?